index.html

```diff
@@ -1,33 +1,155 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <meta charset="UTF-8">
     <link href="./resources/css/index.css" type="text/css" rel="stylesheet">
```

Comment:
all other resources call in this file uses a dash + path structure (e.g.: "/{url}" ). Only this line uses "dot + dash". Try to use the same pattern as the others.